### PR TITLE
Ayyliens can no longer be forced to scream out loud (adds a no_forced scream trait for future use)

### LINKED ISSUE #47219: phobias can cause abductors to speak out loud
https://cdn.discordapp.com/attachments/326826573951008769/635045676325142528/cowardlyabductor.PNG

I had the phobia (supernatural) trauma and the brain damaged language brain trauma as an abductor, and being afraid of my own jumpsuit occasionally forced me to say "AHHHH!!!" out loud (in the brain damaged language) instead of in abductor chat.

Also, according to my experience in this round, you can apparently still be afraid of stuff while wearing a blindfold, although maybe that's because I was wearing the thing that I was afraid of (my own jumpsuit; thanks, random brain trauma event, very cool).

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
webedit so it's most likely going to have issues. I'm sorry for that but I currently not in the position where I can test it
## Why It's Good For The Game
ayys should not be aloud (hahah?) to scream
fixes #47219

## Changelog
:cl: PepperPrepper
fix: ayyliens no longer scream out loud
/:cl:

### CODE REVIEW

**Comment by PepperPrepper** [comment]
That... Should have worked. hmmm

**Comment by optimumtact** [comment]
I'm in half a mind to close this PR, this is exactly why you dont' webedit, github is not an IDE

**Comment by PepperPrepper** [comment]
okay I'm "fairly" sure I've fixed whatever the bots complaining about. If someone is willing to save me from shame and check I'd be grateful.

**Comment by PepperPrepper** [comment]
yeah I'm really sorry about this. @optimumtact I currently can't work on my main pc but I really really had the itch to do something. I'll stick to just working in my branch next time..

**Comment by optimumtact** [comment]
You are mixing tabs and spaces in a file, because of the shitty webeditor

**Comment by PepperPrepper** [comment]
okay I think I've got it. Real sorry about all this.

**Comment by ATH1909** [comment]
Why is this trait tied to the abductor race instead of their tongues (which alien golems also use, by the way)?

**Comment by PepperPrepper** [comment]
Why would it be better to tie it to the tongue? I don't see any benefit.

**Comment by PepperPrepper** [comment]
*alien golems

**Comment by XDTM** [comment]
The tongue is what causes the hive mind speech, so you can have a hiveminded human or a normal-speaking abductor.

**Comment by MCterra10** [comment]
Tbh having it bound to the tongue is pretty cheesable especially for alien golems, it's supposed to balance the fact that you naturally regenerate, but if one of your buddies can remove your tongue and swap in a new one from a monkey in a minute or two, it's pretty wack. Abductor hivemind should be a species trait.

**Comment by XDTM** [comment]
Maybe the tongue is too easy, but I like the idea of having it as an organ, since abductors are surgery-centric. The species itself would be naturally mute without it.

File: code/datums/brain_damage/phobia.dm [hunk 1]
```diff
@@ -119,9 +119,14 @@
 			owner.Stun(70)
 			owner.Jitter(8)
 		if(2)
-			owner.emote("scream")
-			owner.Jitter(5)
-			owner.say("AAAAH!!", forced = "phobia")
+			if(HAS_TRAIT(owner,TRAIT_NO_FORCED_SPEECH))			
+				owner.Jitter(5)
```
Comment (ninjanomnom) [inline code/datums/brain_damage/phobia.dm]: You're doing this either way, take it out of the if/else

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/species_types/golems.dm [hunk 2]
```diff
@@ -252,6 +252,7 @@
 /datum/species/golem/alloy
 	name = "Alien Alloy Golem"
 	id = "alloy golem"
+	inherent_traits = list(TRAIT_RESISTHEAT,TRAIT_NOBREATH,TRAIT_RESISTCOLD,TRAIT_RESISTHIGHPRESSURE,TRAIT_RESISTLOWPRESSURE,TRAIT_NOFIRE,TRAIT_CHUNKYFINGERS,TRAIT_RADIMMUNE,TRAIT_PIERCEIMMUNE,TRAIT_NODISMEMBER,TRAIT_NO_FORCED_SPEECH)
```
Comment (ATH1909) [inline code/modules/mob/living/carbon/human/species_types/golems.dm]: and what if someone put an ayy's tongue in a human?

**Comment by ATH1909** [comment]
> Maybe the tongue is too easy, but I like the idea of having it as an organ, since abductors are surgery-centric. The species itself would be naturally mute without it.

Really? Don't tongueless ayys "speak" (out loud) in the same way that tongueless humans do?

**Comment by PepperPrepper** [comment]
Having a real busy Christmas. I'll reopen this when I have some spare time.

**Pull request closed without merging**